publisher/server/publisher.go

```diff
@@ -115,6 +116,24 @@ func (s *Server) publishContainer(container *docker.APIContainers, ttl time.Dura
 			port := strconv.Itoa(int(p.PublicPort))
 			hostAndPort := s.host + ":" + port
 			if s.IsPublishableApp(containerName) && s.IsPortOpen(hostAndPort) {
+				configKey := fmt.Sprintf("/deis/config/%s/", appName)
+				// check if the user specified a healthcheck URL
+				healthcheckURL := s.getEtcd(configKey + "healthcheck_url")
+				delay, err := strconv.Atoi(s.getEtcd(configKey + "healthcheck_initial_delay"))
+				if err != nil {
+					log.Println(err)
```

Comment:
These should be ignored if the key is unset, but I think it makes sense to log them if they're an invalid integer.